xs/Buffer.xs

```diff
@@ -0,0 +1,53 @@
+MODULE = FFI::Platypus PACKAGE = FFI::Platypus::Buffer
+
+# Stolen from Data::Peek::DGrow
+void
+grow (sv, size, ... )
+    SV     *sv
+    IV      size
+ 
+  PROTOTYPE: $$;$
+  PREINIT:
+    int clear = 1;  
+  PPCODE:
+    if ( items > 2 )
+        clear = SvIV(ST(2));
+
+    if (SvROK (sv))
+        sv = SvRV (sv);
```

Comment:
Question: should we operate on the referent, or error if a reference is passed?  If the former then we should at least document it.  I think I might be in favor of the latter though.

Reply:
The more I think about this the more I think this should be an error.  What if sv is a hashref or a refref etc. ?

Reply:
The current code will force it into a PV, so that indeed might not be a good thing.  I'll change it to croak if it's an RV.